crates/indexer/src/helper.rs

```diff
@@ -39,19 +40,31 @@ impl PolyjuiceArgs {
                 input_size
             ));
         }
-        if args.len() < 52 + input_size as usize {
+
+        let input: Option<Vec<u8>>;
+        let to_address_when_native_transfer: Option<Vec<u8>>;
+        if args.len() == 52 + input_size as usize {
```

Comment:
how about using `match` to remove `else if` ?

Reply:
Using `match` would be like below:

```rust
match args.len() {
  _ if args.len() == 52 + input_size as usize => {

  }
  _ if args.len() == 52 + input_size as usize + 20 => {

  }

  _ => {
     // unknown
  }
}
```

I don't know, both are okay for me. What's your opinion?

Reply:
I think it is more clear to read

Reply:
Done. 1db0157bf7f1d7b2a4b0f9d2f66ddcedf67b5f1a changes to `match` style.